arangod/Scheduler/Scheduler.cpp

```diff
@@ -208,28 +212,76 @@ Scheduler::~Scheduler() {
 }
 
 // do not pass callback by reference, might get deleted before execution
-void Scheduler::post(std::function<void()> const& callback) {
+void Scheduler::post(std::function<void()> const callback, bool isV8,
```

Comment:
If passing `callback` by const ref is not possible, then the comment above should be removed as well.
I am wondering who should delete `callback` here...

Reply:
done